contact_app.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import json
+
+class JSONWriter:
+    def __init__(self):
+        pass
+    def add_contact(self,person):
+        dict = {person.get_phone_number():{"name":person.get_name(),"email":person.get_email_id(),"city":person.get_city()}}
+
+        with open("jsonFile.json","r") as filepointer:
+            if filepointer.read() == "":
+                with open("jsonFile.json", "w") as filepointer:
+                    json.dump(dict, filepointer)
+            else:
+                filepointer.seek(0)
+                data = json.load(filepointer)
+                if str(person.get_phone_number()) in data.keys():
+                    print("person already exist")
+                else:
+                    data.update(dict)
+                    with open("jsonFile.json", "w") as filepointer:
+                        filepointer.write(json.dumps(data))
+
+    def delete_contact(self,number):
+        with open("jsonFile.json", "r") as filepointer:
+            if filepointer.read() == "":
+                print "Directory is empty"
```

Comment:
JSONWriter is the Persistence Layer and it should not assume that the app is a Console App. So NO prints should be used
 Add a line note